core/src/main/resources/defaults/blueprints/cdp-streaming-small-710.bp

```diff
@@ -0,0 +1,125 @@
+{
+  "description": "CDP 1.3 - Streams Messaging Cluster (small)",
+  "blueprint": {
+    "cdhVersion": "7.1.0",
+    "displayName": "CDP 1.3 - Streams Messaging Cluster (small)",
+    "services": [
+      {
+        "refName": "streams_messaging_manager",
+        "serviceType": "STREAMS_MESSAGING_MANAGER",
+        "serviceConfigs": [
+          {
+            "name": "zookeeper_service",
+            "ref": "zookeeper"
+          },
+          {
+            "name": "cm.metrics.port",
+            "value": "7183"
+          },
+          {
+            "name": "cm.metrics.protocol",
```

Comment:
do we need this?

Reply:
yes

Reply:
Latest PR addresses it by a Config Provider, logged JIRA for long term solution based on CSD framework.